articles/azure-resource-manager/bicep/best-practices.md

```diff
@@ -105,7 +105,7 @@ For more information about Bicep variables, see [Variables in Bicep](variables.m
 
 ## Outputs
 
-* Make sure you don't create outputs for sensitive data. Output values can be accessed by anyone who has access to the deployment history. They're not appropriate for handling secrets.
+* Make sure you don't create outputs for sensitive data unless they're marked as secure. Output values can be accessed by anyone who has access to the deployment history. To protect secrets, use the @secure() decorator (available in Bicep v0.35.1 and later) to prevent sensitive outputs from being logged or displayed.
```

Comment:
```suggestion
* Mark sensitive data in outputs by using the [@secure() decorator](outputs.md#secure-outputs), which prevents sensitive outputs from being logged or displayed. Otherwise, output values can be accessed by anyone who has access to the deployment history.
```